dissect/target/plugins/os/windows/notifications.py

```diff
@@ -63,11 +198,149 @@ def __init__(self, target):
                 self.wpndb_files.append((user_details.user, wpndb_file))
 
             if appdb_file.exists():
-                self.appdb_files.append((user_details.user, appdb_file))
+                with appdb_file.open(mode="rb") as fp:
+                    chunk = c_appdb.Chunk(fp)
+                    if chunk.Header.Magic == APPDB_MAGIC and chunk.Header.Version == 3:
```

Comment:
Maybe add a warning for unsupported versions and list the specified version number?